iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/relation/citation/CitationRelationExporterJob.java

```diff
@@ -54,8 +54,8 @@ public static void main(String[] args) {
 
     @Parameters(separators = "=")
     private static class JobParameters {
-        @Parameter(names = "-isSparkSessionManaged", arity = 1)
-        private Boolean isSparkSessionManaged = Boolean.TRUE;
+        @Parameter(names = "-sharedSparkSession")
```

Comment:
If we agree on changing the variable name then we should adjust the parameter name in the same way.